lsp-mode.el

```diff
@@ -235,7 +235,13 @@ modifiers into account. Only applies if
 (defvar-local lsp--cur-workspace nil)
 
 (defvar-local lsp--cur-version 0)
+(defvar-local lsp--virtual-buffer-connections nil)
+(defvar-local lsp--virtual-buffer nil)
+(defvar lsp--virtual-buffer-mappings (ht))
 
+(defvar-local lsp--virtual-buffer-connections nil)
+(defvar-local lsp--virtual-buffer nil)
+(defvar lsp--virtual-buffer-mappings (ht))
```

Comment:
These three lines are duplicated.

Reply:
Thank you, fixed. Apparently I have pasted them twice.